velox/exec/MemoryReclaimer.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Why don't we call this file DefaultMemoryReclaimer.h and put it in parallel with memory/MemoryReclaimer.h? Or put this class inside memory/MemoryReclaimer.h?